tools/shared.py

```diff
@@ -180,9 +180,9 @@ def check_call(cmd, *args, **kw):
   try:
     return run_process(cmd, *args, **kw)
   except subprocess.CalledProcessError as e:
-    exit_with_error("'%s' failed (%d)", ' '.join(cmd), e.returncode)
+    exit_with_error("'%s' failed (%d)", ' '.join(e.cmd), e.returncode)
```

Comment:
what's the difference between `cmd` and `e.cmd`?

Reply:
Nothing really in this case.  e.cmd is available whenever you catch this exception.   cmd just happens to be a local here too and should be identical.